### PR TITLE
Build game ci

### PR DESCRIPTION
**Pull request by zicons**

### CODE REVIEW

File: Assets/Editor/Build/BuildScript.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        {
+            Dictionary<string, string> options = GetValidatedOptions();
+
+            string increment = options["increment"];
+            if (string.IsNullOrEmpty(increment))
+                increment = "build";
+
+            string buildTag = options["buildTag"];
+            if (string.IsNullOrEmpty(buildTag))
+                buildTag = "gitlab_tag";
```
Comment (SomethingWittyWithATechAspect) [inline Assets/Editor/Build/BuildScript.cs:29]: gitlab_tag?

File: Assets/Editor/Build/BuildScript.cs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+
+            if (summary.result == BuildResult.Succeeded)
+            {
+                Debug.Log("Build succeeded: " + summary.totalSize + " bytes");
+            }
+
+            if (summary.result == BuildResult.Failed)
+            {
+                Debug.Log("Build failed");
```
[the remark this answers is not in the document]
Comment (SomethingWittyWithATechAspect) [reply Assets/Editor/Build/BuildScript.cs:56]: Or Console.log with out to text file

**Comment by SomethingWittyWithATechAspect** [comment]
maybe use Console.log everwhere and write output to logfile instead of console since it is supposed to run remote

File: Assets/Editor/Build/BuildScript.cs (hunk 1 above)
Comment (zicons) [reply Assets/Editor/Build/BuildScript.cs:29]: removed this check and added it bellow where all the inputs are getting checked

File: Assets/Editor/Build/BuildScript.cs [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+            if (string.IsNullOrEmpty(buildTag))
+                buildTag = "gitlab_tag";
+
+            string buildVersion = options["buildVersion"];
+            if (string.IsNullOrEmpty(buildTag))
+                buildVersion = "none";
+
+            string buildPath = options["customBuildPath"];
+            if (string.IsNullOrEmpty(buildPath))
+                buildPath = @"C:\bob\BoB.exe";
```
[the remark this answers is not in the document]
Comment (zicons) [reply Assets/Editor/Build/BuildScript.cs:37]: removed this check and added it bellow where all the inputs are getting checked

File: Assets/Editor/Tests/CameraTest.cs [hunk 4]
```diff
@@ -39,4 +38,4 @@ public void TearDown()
     {
         _altUnityDriver.Stop();
     }
-}
+}
```
Comment (SomethingWittyWithATechAspect) [inline Assets/Editor/Tests/CameraTest.cs:41]: eof nl

**Review by SomethingWittyWithATechAspect — APPROVED** [review]

**Comment by SomethingWittyWithATechAspect** [comment]
wrong button

**Pull request merged**